portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFunFact() {
+  const funFacts =
+      ['Favorite TV show is Game of Thrones', 'Favorite Sport is Football', 'I have fractured every finger on my left hand', 'I have played the double bass since Third Grade'];
```

Comment:
You might reformat this so the line isn't long. Google javascript style guide (and most Google style guides) recommend [80 char max limit](https://google.github.io/styleguide/jsguide.html#formatting-column-limit) per line unless there is a functional reason not to break it.